cms/djangoapps/contentstore/management/commands/delete_course.py

```diff
@@ -57,7 +57,11 @@ def add_arguments(self, parser):
     def handle(self, *args, **options):
         try:
             # a course key may have unicode chars in it
-            course_key = text_type(options['course_key'], 'utf8')
+            try:
+                course_key = text_type(options['course_key'], 'utf8')
```

Comment:
What does this decoding support? The possible sending of encoded Unicode chars into course_key? On my LMS shard 4 branch, I fixed this same error by just dropping the `, 'utf8'`. This change is safer but is it necessary?

Reply:
Seems like it was added intentionally to cover that case: https://github.com/edx/edx-platform/commit/8bc7ae146690d05321cdeeaef7694afd85789d55#diff-6e2de8bda28b18a6d50bc686d199e81bL53